projects/kit/src/lib/masks/time/utils/tests/stringify-time.spec.ts

```diff
@@ -0,0 +1,66 @@
+import {describe, expect, it} from '@jest/globals';
+import type {MaskitoTimeMode} from '@maskito/kit';
+import {maskitoStringifyTime} from '@maskito/kit';
+
+describe('maskitoStringifyTime', () => {
+    const testCases = new Map<MaskitoTimeMode, Array<{text: string; ms: number}>>([
+        [
+            'HH:MM:SS.MSS',
+            [
+                {text: '00:00:00.000', ms: 0},
```

Comment:
The same comment (order is `from => to`)

```suggestion
                {ms: 0, text: '00:00:00.000'},
```

Reply:
Done.